docs/swagger/swagger.yaml

```
@@ -584,451 +420,495 @@ components:
         - body
     Response:
       allOf:
-        - $ref: '#/components/schemas/NewResponse'
-        - type: object
-          properties:
-            modified_at:
-              type: string
-              format: date-time
-          required:
-            - modified_at
-    ResponseDetails:
-      allOf:
-        - $ref: '#/components/schemas/NewResponse'
+        - $ref: "#/components/schemas/QuestionnaireID"
         - type: object
           properties:
             responseID:
               type: integer
               example: 1
           required:
             - responseID
-    ResponseSummary:
+        - $ref: "#/components/schemas/NewResponse"
+    Responses:
+      type: array
+      items:
+        $ref: "#/components/schemas/Response"
+    ResponseBody:
+      oneOf:
+        - $ref: "#/components/schemas/ResponseSettingsText"
+        - $ref: "#/components/schemas/ResponseSettingsTextLong"
+        - $ref: "#/components/schemas/ResponseSettingsNumber"
+        - $ref: "#/components/schemas/ResponseSettingsSingleChoice"
+        - $ref: "#/components/schemas/ResponseSettingsMultipleChoice"
+        - $ref: "#/components/schemas/ResponseSettingsScale"
+    ResponseSettingsText:
       type: object
       properties:
-        responseID:
-          type: integer
-          example: 1
-        questionnaireID:
-          type: integer
-          example: 1
-        questionnaire_title:
-          type: string
-          example: 第1回集会らん☆ぷろ募集アンケート
-        description:
-          type: string
-          example: 2017年度入学学部生
-        res_time_limit:
+        text:
           type: string
-          format: date-time
-        submitted_at:
-          type: string
-          format: date-time
-        modified_at:
+    ResponseSettingsTextLong:
+      type: object
+      properties:
+        textLong:
           type: string
-          format: date-time
-      required:
-        - responseID
-        - questionnaireID
-        - questionnaire_title
-        - modified_at
-    ResponseBody:
+    ResponseSettingsNumber:
+      type: object
+      properties:
+        number:
+          type: number
+    ResponseSettingsSingleChoice:
+      type: object
+      properties:
+        index:
+          type: integer
+          description: |
+            選択肢のインデックス
+    ResponseSettingsMultipleChoice:
+      type: object
+      properties:
+        answerIndexes:
+          type: array
+          items:
+            type: integer
+          description: |
+            選択肢のインデックスの配列
+    ResponseSettingsScale:
       type: object
       properties:
-        questionID:
+        number:
+          type: integer
+    Result:
+      type: object
+      properties:
+        questionnaireID:
           type: integer
           example: 1
-        question_type:
-          $ref: '#/components/schemas/QuestionType'
-        response:
-          type: string
-          example: リマインダーBOTを作った話
-        option_response:
+        responseNum:
```

回答の合計数を指してるなら修正して欲しいです

```suggestion
        responseCount:
```

---

修正しました！